src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -17,16 +18,35 @@ type Props = InjectedProps & {
   tabBarPosition?: 'top' | 'bottom',
   tabBarComponent?: React.ComponentType<*>,
   tabBarOptions?: TabBarOptions,
+  lazyOnSwipe: boolean,
+  sceneAlwaysVisible: boolean,
```

Comment:
How is the performance during lazy load? The example app is really simple, but since it's doing `setState` during swipe gestures I'm expecting a visible performance drop. Have you tried it in an app with more complex content in tabs?